tools/c7n_tencentcloud/c7n_tencentcloud/resources/cam.py

```diff
@@ -0,0 +1,460 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from concurrent.futures import as_completed
+import copy
+import json
+import pytz
+from c7n.exceptions import PolicyValidationError
+from c7n.utils import chunks, type_schema
+from c7n_tencentcloud.provider import resources
+from c7n_tencentcloud.query import ResourceTypeInfo, QueryResourceManager, DescribeSource
+from c7n_tencentcloud.utils import isoformat_datetime_str, PageMethod, convert_date_str
+from c7n.filters import ValueFilter, Filter
+
+
+class DescribeCAM(DescribeSource):
+    """DescribeCAM"""
+    def get_resource_qcs(self, resources):
+        qcs_list = []
+
+        # for CAM, there is no region in qcs
+        for r in resources:
+            qcs = self.get_qcs(self.resource_type.service,
+                               "",
+                               self.resource_manager.config.account_id,
+                               self.resource_type.resource_prefix,
+                               r[self.resource_type.id])
+            qcs_list.append(qcs)
+        return qcs_list
+
+
+@resources.register("cam-user")
+class User(QueryResourceManager):
+    """
+    User - user management in Cloud Access Management (CAM)
+    https://www.tencentcloud.com/document/product/1021/37656
+    """
+    source_mapping = {"describe": DescribeCAM}
+
+    class resource_type(ResourceTypeInfo):
+        """resource_type"""
+        id = "Uin"
+        endpoint = "cam.tencentcloudapi.com"
+        service = "cam"
+        version = "2019-01-16"
+        enum_spec = ("ListUsers", "Response.Data[]", {})
+        taggable = True
+        resource_prefix = "uin"
+        batch_size = 10
+        datetime_fields_format = {
+            "CreateTime": ("%Y-%m-%d %H:%M:%S", pytz.timezone("Asia/Shanghai")),
+            "LastLoginTime": ("%Y-%m-%d %H:%M:%S", pytz.timezone("Asia/Shanghai")),
+            "access_keys.CreateTime": ("%Y-%m-%d %H:%M:%S", pytz.timezone("Asia/Shanghai")),
+            "access_keys.LastUsedDate": ("%Y-%m-%d", pytz.timezone("Asia/Shanghai"))
+        }
+
+    def __init__(self, ctx, data):
+        super().__init__(ctx, data)
+        self._query_client = None
+
+    @property
+    def client(self):
+        """client"""
+        if self._query_client is None:
+            self._query_client = self.get_client()
+        return self._query_client
+
+    def augment(self, resources):
+        """augment"""
+        for item in resources:
+            fm = self.resource_type.datetime_fields_format["CreateTime"]
+            item["CreateTime"] = isoformat_datetime_str(item["CreateTime"], fm[0], fm[1])
+        return resources
+
+
+@User.filter_registry.register('group')
+class GroupMembership(ValueFilter):
```

Comment:
can we add a docstring here for the group filter

Reply:
1. added some doctoring: official doc url
2. just added one comment to clarify that "one sub-account can only belong to 10 user groups at most"